.babelrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
sposób w jaki używasz babela sprawia że plik `babelrc` nie jest obsługiwany. 

Sposób z dodawaniem go do HTML jest dobry w środowisku deweloperskim ale nie powinno się tak robić w aplikacji produkcyjnej.

Możesz spróbować użyć babela tak jak było to opisane w module czyli zainstalowanie go przy użyciu npm i później dodanie odpowiedniego taska w package.json

Jedyne co będziesz musiał zmienić względem tego co jest opisane w module to ścieżka jakiej używasz w `build` tasku. W module opis wskazuje że cały kod JS znajduje się w jednym pliku, u Ciebie tak nie jest. Żeby uwzględnić kod JS ze wszystkich katalogów task build powinien wyglądać w następujący sposób:
`"build": "babel **/*.js --watch --out-file build.js"`